.circleci/config.yml

```diff
@@ -18,13 +18,25 @@ var_2: &install-dependencies
       # install git and bash
       nix-env -f '<nixpkgs>' -i -A gitAndTools.git -A bash
 
+var_3: &install-cachix
+  run:
+    name: Install Cachix
+    shell: /usr/bin/env bash -euxo pipefail
```

Comment:
```suggestion
```

I don't think you need that as it's already set to `bash` with the `-e` option: https://circleci.com/docs/2.0/configuration-reference/#default-shell-options

Reply:
https://app.circleci.com/jobs/github/kalbasit/shabka/5270
Actually it doesn't find bash in `/bin/bash` so it falls back to `/bin/sh`